src/components/RuleCard/index.jsx

```diff
@@ -682,6 +698,15 @@ function RuleCard({ rule, onRuleDelete, readOnly, ...props }) {
             )}
           </Fragment>
         )}
+        {requiresSignoff && (
+          <Fragment>
+            <div className={classes.space} />
```

Comment:
Instead of adding an empty div with a class, allow the SignoffSummary to handle a `className`:

```jsx
// SignoffSummary/index.jsx

import classNames from 'classnames';
import { object } from 'prop-types';

// ...

function SignoffSummary(props) {
  const { requiredSignoffs, signoffs, className } = props;

  // ...

  <div className={classNames(classes.listWrapper, className)}>
    // ...
  </div>
}


SignoffSummary.propTypes = {
  // ...
  className: object,
};

SignoffSummary.defaultProps = {
  className: null,
};
```